modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/image_gallery_display/view_images.jsp

```diff
@@ -91,15 +91,17 @@ DLPortletInstanceSettingsHelper dlPortletInstanceSettingsHelper = new DLPortletI
 					thumbnailSrc = DLURLHelperUtil.getThumbnailSrc(fileEntry, fileVersion, themeDisplay);
 				}
 
+				String description = fileEntry.getDescription();
```

Comment:
Hey @AliciaGarciaGarcia , 

What do you mean by "sorted by the use"?

I added just before `title` definition since it is used before title variable later on.

Reply:
I don't understand it either... let's forward it and see if Brian is ok.